nautobot/project-static/css/hide_body.css

```diff
@@ -0,0 +1,14 @@
+/* This CSS code cannot go into base.css.
+It is loaded immediately in the <head> section before anything else.
```

Comment:
Still a bit curious about this. `base.css` is also loaded in the `<head>` of the file. Are we just relying on the order of declarations in `<head>` to ensure that this file gets loaded early enough that we avoid the white flash? Would it be more reliable to embed this CSS directly in the page template itself rather than linking it as a separate file for the browser to load?

Reply:
No specific reason it is in a separate file, other than that's how the other CSS files are loaded (via separate files).
I do not know if it would be more reliable, but I am ok with embedding it directly if desired.

Yes the ordering is important, as we want to hide the `<body>` contents right away, before anything else starts to load.  Then we render `<body>` after everything is loaded, which helps with the brief switchover from white/base CSS (server side) to when JS loads dark mode CSS (client side).